src/EA.Weee.Domain/Evidence/EvidenceNoteFilter.cs

```diff
@@ -28,6 +28,8 @@ public class EvidenceNoteFilter
 
         public DateTime? EndDateSubmitted { get; set; }
 
+        public int? NoteTypeFilter { get; set; }
```

Comment:
As above I think this should be mandatory unless we use it somewhere else that doesnt require a note type